apps/wallet/src/ui/app/components/menu/content/ConnectLedgerModal.tsx

```diff
@@ -0,0 +1,96 @@
+import { ModalDialog } from '_src/ui/app/shared/ModalDialog';
+import { Text } from '_src/ui/app/shared/text';
+
+type ConnectLedgerModalProps = {
+    isOpen: boolean;
+    onClose: () => void;
+    onConfirm: () => void;
+};
+
+export function ConnectLedgerModal({
+    isOpen,
+    onClose,
+    onConfirm,
+}: ConnectLedgerModalProps) {
+    return (
+        <ModalDialog
+            isOpen={isOpen}
+            title="Connect Ledger Wallet"
+            onClose={onClose}
+            body={
+                <div className="flex flex-col items-center">
+                    <div className="mt-4.5">
+                        <LedgerLogo />
+                    </div>
+                    <div className="break-words text-center mt-4.5">
+                        <Text variant="p2" color="steel-darker" weight="normal">
+                            Connect your ledger to your computer, unlock it, and
+                            launch the Sui app. Click Continue when done.
+                        </Text>
+                        <div className="mt-2">
+                            <Text
+                                variant="p2"
+                                color="steel-dark"
+                                weight="normal"
+                            >
+                                <span>Need more help?</span>{' '}
+                                {/* TODO: Replace this link with the correct URL once the tutorial is ready */}
+                                <ExternalLink
+                                    href="https://mystenlabs.com"
+                                    className="text-hero-dark"
+                                    showIcon={false}
+                                >
+                                    View tutorial.
+                                </ExternalLink>
+                            </Text>
+                        </div>
+                    </div>
+                </div>
+            }
+            footer={
+                <div className="flex flex-row self-center gap-3">
+                    <div>
+                        <Button
+                            variant="outline"
+                            text="Cancel"
+                            onClick={onClose}
+                        />
+                    </div>
+                    <div>
+                        <Button
+                            variant="outline"
+                            text="Continue"
+                            onClick={onConfirm}
+                        />
+                    </div>
+                </div>
+            }
+        />
+    );
+}
+
+// TODO: We should probably use a loader like @svgr/webpack so that we can provide SVG files
+// and have them be automatically importable in React components. From playing around with
+// this, there seems to be an issue where TypeScript bindings aren't correctly generated
+// (see https://github.com/gregberge/svgr/pull/573)
+function LedgerLogo() {
```

Comment:
`core/icons` should be the place for this. There should be some instructions how to add more icons

Reply:
Hmm this isn't really an icon though, it's an asset that's specific to the wallet which was why I didn't put it there

Reply:
Yeah this is kind of tricky. I could see the argument for `core/icons` and I could see the argument for inline in the wallet. Rather than baking this into code, we could also just keep an `.svg` file and load it in an `img` tag or something. Probably need some strategy here for "assets that are not icons". But for now this is fine.

Reply:
+1 about having a more standardized approach to assets that aren't icons. Fortunately, there aren't many of these assets in the wallet today so we can punt this down the road for now  :)